_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
+The license pop-up now lacks the ability to just disappear when you change orientation. This was fixed by [@nmurali94](https://github.com/nmurali94).
+[@budde92](https://github.com/budde25) clicked enough keyboard to where you can now delete a file you just downloaded _and_ re-download it _without_ the app crashing. In what is a conundrum, we are not sure why you'd want to before now, but do.
 
-[@mhmdanas](https://github.com/mhmdanas) changed a code call because its origin was redundant. Yay for less redundancy. When auto-queue is enabled, Newpipe wants to add a stream to the queue the moment you open a video. But if there are no video suggestions on that page, it causes a crash. This has been fixed by [@wb9688](https://github.com/wb9688).
+[@mhmdanas](https://github.com/mhmdanas) changed a code call because its origin was redundant. Less redundancy good. When auto-queueing takes place, Newpipe wants to add a stream to the queue the moment you open a video. If no video suggestions were to be found on that page, that caused a crash. And then [@wb9688](https://github.com/wb9688) put an end to such shenanigans.
 
 ## Development
 
-Development. Where all the magic which keeps this app running happens. It saw some improvements in this release: (Warning: Nerd talk ahead)
+Development. The nerd magic sourced from the purest of autism. Embrace yourself:
 
 - The extractor will ignore OTF streams now, since they don't work with progressive HTTP, and the extractor can't work with DASH just yet. This has been a major point of inconvenience for a lot of users (and us, too!) so we're eager to mitigate the problem until the proper fix sometime later. ([@wb9688](https://github.com/wb9688))
 
-- A Checkstyle rule was added so it would show an error when a local variable which could be `final` wasn't declared as such. ([@XiangRongLin](https://github.com/XiangRongLin))
+- A [Checkstyle](https://checkstyle.sourceforge.io/) rule was added to show an error when a local variable which could be `final` wasn't declared as such. ([@XiangRongLin](https://github.com/XiangRongLin))
 
-- A library we use, PrettyTime, has been updated to a newer version so that a workaround can be removed from NewPipe's code. We get a performance improvement as a bonus! ([@B0pol](https://github.com/B0pol))
+- A library we use, PrettyTime, has been updated to a newer version so that a workaround can be removed from NewPipe's code. The performance improvement is a bonus! ([@B0pol](https://github.com/B0pol))
 
-- Numbers and uppercase letters are now allowed in the application ID. ([@Stypox](https://github.com/Stypox))
+- Numbers and uppercase letters are now allowed in the app ID. ([@Stypox](https://github.com/Stypox))
 
-- Contributors need to discuss their contributions now prior to opening a PR. ([@gkeegan](https://github.com/gkeegan))
```

Comment:
That doesn't sound good, Verb objective subjective shuffled.

Reply:
> Verb objective subjective shuffled.

[Subject] needs to discuss [subject]'s [object]. What is the problem?

Reply:
The problem is it isn't how English works. It is "contributor*" twice.